documentation/modules/exploit/linux/http/nagios_xi_configwizards_authenticated_rce.md

```diff
@@ -0,0 +1,75 @@
+## Vulnerable Application
+This module exploits CVE-2021-25926, CVE-2021-25927, or CVE-2021-25928, OS command injection vulnerabilities in `/nagiosxi/config/monitoringwizard.php` that enables an authenticated user to achieve remote code execution on NagiosXI 5.7.5. There are two vulnerable parameters (ip_address, plugin_output_len), and either can be used to achieve code execution.
+
+The module's `check` method takes advantage of the `Msf::Exploit::Remote::HTTP::NagiosXi` mixin in order to authenticate to the target and
+obtain the Nagios XI version number, which is then used to check if the target is version 5.7.5 and therefore vulnerable.
+
+An OVA with the vulnerable NagiosXI application can be downloaded from Nagios [here](https://assets.nagios.com/downloads/nagiosxi/5/ovf/nagiosxi-5.7.5-64.ova).
+
+
+## Verification Steps
+
+
+- [ ] Start `msfconsole`
+- [ ] `use exploit/linux/http/nagios_xi_configwizards_authenticated_rce`
+- [ ] `set RHOSTS TARGET_IP`
+- [ ] `set RPORT 443`
+- [ ] `set SSL true`
+- [ ] `set USERNAME USER`
+- [ ] `set PASSWORD PASSWORD`
+- [ ] `set TARGET_URL_PARAM plugin_output_len`
+- [ ] `set LHOST YOUR_IP`
+- [ ] `set LPORT YOUR_LISTENING_PORT`
+- [ ] `run`
+
+
+## Options
+
+### USERNAME
+A valid NagiosXI username, which can be for an administrator or regular user.
+
+### PASSWORD
+The password for the provided NagiosXI username.
+
+### TARGET_URL_PARAM
+The vulnerable URL parameter to inject payloads into. Defaults to plugin_output_len with ip_address as another option.
```

Comment:
```suggestion
The vulnerable URL parameter to inject payloads into. Defaults to `plugin_output_len`.
```

Don't think the extra info is needed here, this should be sufficient.

Reply:
This was removed in favor of TARGET_CONFIG_WIZARD with the recent updates which looks fine in its current state. Marking this as resolved.